webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -2068,6 +2105,44 @@
          rdfs:label "Transkription"@de ,
                     "Transcription"@en .
 
+### ###########################################
+### beol:comment
+
+:entryComment rdf:type owl:Class ;
+
+         rdfs:subClassOf knora-base:Resource ,
+
+         [
+           rdf:type owl:Restriction ;
+           owl:onProperty :hasText ;
+           owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+           salsah-gui:guiOrder "0"^^xsd:nonNegativeInteger
+         ] ,
+         [
+           rdf:type owl:Restriction ;
+           owl:onProperty :seqnum ;
+           owl:cardinality "1"^^xsd:nonNegativeInteger ;
+           salsah-gui:guiOrder "1"^^xsd:nonNegativeInteger
+         ] ,
+         [
+           rdf:type owl:Restriction ;
+           owl:onProperty :entryCommentOf ;
+           owl:cardinality "1"^^xsd:nonNegativeInteger ;
```

Comment:
I think it can be the case that one comment can be used for multiple Meditatio entries. Then maybe this should be minCardinality=1.

Reply:
Currently, this is not the case. Commentaries refer to one Meditatio only.